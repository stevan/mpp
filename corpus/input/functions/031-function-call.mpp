my $sum = add(5, 10);