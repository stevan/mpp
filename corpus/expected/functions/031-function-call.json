[
  {
    "type": "Declaration",
    "declarator": "my",
    "variable": {
      "type": "Variable",
      "name": "$sum"
    },
    "initializer": {
      "type": "Call",
      "name": "add",
      "arguments": [
        {
          "type": "Number",
          "value": "5"
        },
        {
          "type": "Number",
          "value": "10"
        }
      ]
    }
  }
]